stl/inc/__msvc_int128.hpp

```diff
@@ -9,12 +9,25 @@
 
 #include <yvals_core.h>
 #if _STL_COMPILER_PREPROCESSOR
-#ifdef __cpp_lib_concepts
+#include <cstdint>
+#include <intrin.h> // TRANSITION, GH-2520
+#include <limits>
+#include <type_traits>
+
+#if _HAS_CXX20
 #include <bit>
 #include <compare>
+#define _ZERO_OR_NO_INIT
+#else // ^^^ _HAS_CXX20 / vvv !_HAS_CXX20
+#define _ZERO_OR_NO_INIT = {0} // Trivial default initialization is not allowed in constexpr functions before C++20.

```

Comment:
Is there any reason not to do

```suggestion
#define _ZERO_OR_NO_INIT {} // Trivial default initialization is not allowed in constexpr functions before C++20.
```

or

```suggestion
#define _ZERO_OR_NO_INIT = {}
```
? The `0` doesn't seem to provide anything to me.

Reply:
Direct-list-initialization is used now.